docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -722,6 +722,55 @@ directly or used in conditional statements.
 * ``$file_size``
     Size of the file for this row.
 
+Functions
+---------
+
+The connector exposes following table functions:
+
+system.table_changes
+^^^^^^^^^^^^^^^^^^^^
+
+Allows reading Change Data Feed (CDF) entries. This feature allows
+Delta tables to track row-level changes between versions of a Delta table.
+When enabled on a Delta table, it records "change events" for all the data
```

Comment:
```suggestion
When <insert actual table property name here> is enabled on a Delta table, the connector records "change events" for all the data
```

Reply:
the property name is `change_data_feed_enabled` so a sentence like 
```
When change_data_feed_enabled is enabled on a Delta table, the connector records "change events" for all the data
```
will look weird to me

Reply:
what about 
```
When change_data_feed_enabled is set to true on a Delta table, the connector records "change events" for all the data
```

Reply:
How about 

```
When the ``change_data_feed_enabled`` table property is set to ``true`` on a specific Delta Lake table, the connector records change events for all data changes on the table. 
```